tests/ignite/contrib/metrics/regression/test_canberra_metric.py

```diff
@@ -63,7 +63,7 @@ def test_compute():
     assert canberra.pairwise([v1, v2])[0][1] == pytest.approx(np_sum)
 
 
-def _test_distrib_compute(device):
+def _test_distrib_compute(device, tol=1e-4):
```

Comment:
Why 1e-4 by default ? Let's put it a bit stricter 1e-8

Reply:
Ok. It was the value in ssim metric, I did copy/paste. So 1.e-8 and 1.e-3 when tpu ?

Reply:
Let me run it on TPUs

Reply:
Looks like canberra tests pass without any modifications. The problem is with R2 only